composer.json

```
@@ -22,7 +22,7 @@
         "m6web/php-cs-fixer-config": "^1.0",
         "atoum/atoum":      "@stable",
         "m6web/redis-mock": "@stable",
-        "symfony/symfony": "~2.3||~3.0||~4.0",
+        "symfony/symfony": "~4.0",
```

```suggestion
        "symfony/symfony": "~4.2",
```

because `TreeBuilder::getRootNode()` appeared in SF 4.2 
=> https://github.com/symfony/symfony/blob/4.2/src/Symfony/Component/Config/Definition/Builder/TreeBuilder.php#L57